src/components/AdminPage/CategoryList.module.css

```diff
@@ -3,6 +3,7 @@
 }
 
 .Form {
- position: sticky;
- top: 0px;
+  /* position: sticky; */
```

Comment:
rather than leaving these commented out I would suggest just removing them in order to keep things clean

Reply:
OK will do